exporters/stats/stackdriver/README.md

```diff
@@ -157,4 +157,12 @@ Stackdriver exporter adds a new `Metric` label for each custom `Metric` to ensur
 of the `Timeseries`. The format of the label is: `{LANGUAGE}-{PID}@{HOSTNAME}`, if `{PID}` is not
 available a random number will be used.
 
+### Why did I get an error "java.lang.NoSuchMethodError: com.google.common.base.Throwables.throwIfInstanceOf"?
```

Comment:
Should this heading be more general, to handle other possible uses of new Guava methods?

Reply:
Make sense, updated.